linux_os/guide/services/ftp/ftp_configure_vsftpd/ftp_present_banner/rule.yml

```diff
@@ -44,3 +44,5 @@ ocil: |-
     <pre>$ sudo grep "banner_file" /etc/vsftpd/vsftpd.conf
     {{% endif %}}
     banner_file=/etc/issue</pre>
+
+{{{ warning_rule_deprecated_cause("SLES 12 STIG Revision v2R10", release='0.1.69') | indent(4) }}}
```

Comment:
This deprecation cause is specific to SLES 12 STIG. I think it should be included in a Jinja2 conditional other wise the warning will be visible in other products which may use this rule. If so, this will confuse the users.

Reply:
Make sense but can you please suggest a deprecation clause for rhel7, since it seems that I need to add a deprecation clause anyways but the rule was valid for sle12 and rhel7 as far as I see from the identifiers